src/css/input.css

```diff
@@ -118,6 +123,36 @@ article a[href*="//"]:not([href*="defichainwiki.com"])::after
   content: "" !important;
 }
 
+.formula_bubble {
+  background: linear-gradient(0deg, #fc1dae 21.58%, #00f6ff 103.79%);
+  border-radius: 10px;
+  padding: 6px 6px 6px 6px;
+}
+
+.background_map {
+  background-image: url("/static/img/map_light.svg");
+  border-radius: 10px;
+  padding: 0px 0px 0px 0px;
+}
+
+.formula_map {
+  background-image: url("/static/img/map_light.svg");
+  border-radius: 10px;
+  padding: 30px 30px 10px 30px;
+}
+
+.formula_map_bold {
```

Comment:
Where is this needed?

Reply:
Nowhere, it was used in one of the version of solving backgrounds for numeric examples.  It should be removed.